NEXUS_GLOBAL_VERSION

```diff
@@ -1 +1 @@
-2.2.0
+3.0.0
```

Comment:
Updated!

Reply:
I see you added a `NEXUS_MODULE_VERSION_STRING` -- to be consistent with the embedded modules, I was thinking this would be a MODULE_VERSION file. Why the difference?